openlibrary/templates/account/reading_log.html

```diff
@@ -71,7 +71,21 @@
       $:macros.Pager(int(current_page), doc_count, results_per_page=results_per_page)
   $else:
     <ul class="list-books">
-      $if checkin_year:
+      $if q:
+        <form method="GET" class="olform pagesearchbox">
+          <input type="text" minlength="3" placeholder="$_('Search your reading log')" name="q" value="$(query_param('q', ''))"/>
+          <input type="submit"/>
+        </form>
+        $ query = query_param('q')
+        <center>
```

Comment:
```suggestion
        <div class="center">
```
The `<center>` element has been deprecated since HTML 4.